sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -315,3 +315,196 @@ ALTER TABLE `form_care_plan` CHANGE `date` `date` DATETIME NULL DEFAULT NULL;
 #IfMissingColumn api_token context
 ALTER TABLE api_token ADD COLUMN `context` TEXT COMMENT 'context values that change/govern how access token are used';
 #EndIf
+
+#IfRow2D list_options list_id language notes eng
+CREATE TEMPORARY TABLE lang_updates_610 (notes char(15), twodigit_notes char(2));
+START TRANSACTION;
+INSERT INTO lang_updates_610 VALUES
+('abk','ab'), 
+('aar','aa'), 
+('afr','af'), 
+('aka','ak'), 
+('alb(B)|sqi(T)','sq'), 
+('amh','am'), 
+('ara','ar'), 
+('arg','an'), 
+('arm(B)|hye(T)','hy'), 
+('asm','as'), 
+('ava','av'), 
+('ave','ae'), 
+('aym','ay'), 
+('aze','az'), 
+('bam','bm'), 
+('bak','ba'), 
+('baq(B)|eus(T)','eu'), 
+('bel','be'), 
+('ben','bn'), 
+('bih','bh'), 
+('bis','bi'), 
+('nob','nb'), 
+('bos','bs'), 
+('bre','br'), 
+('bul','bg'), 
+('bur(B)|mya(T)','my'), 
+('cat','ca'), 
+('khm','km'), 
+('cha','ch'), 
+('che','ce'), 
+('nya','ny'), 
+('chi(B)|zho(T)','zh'), 
+('chu','cu'), 
+('chv','cv'), 
+('cor','kw'), 
+('cos','co'), 
+('cre','cr'), 
+('hrv','hr'), 
+('cze(B)|ces(T)','cs'), 
+('dan','da'), 
+('div','dv'), 
+('dut(B)|nld(T)','nl'), 
+('dzo','dz'), 
+('eng','en'), 
+('epo','eo'), 
+('est','et'), 
+('ewe','ee'), 
+('fao','fo'), 
+('fij','fj'), 
+('fin','fi'), 
+('fre(B)|fra(T)','fr'), 
+('ful','ff'), 
+('gla','gd'), 
+('glg','gl'), 
+('lug','lg'), 
+('geo(B)|kat(T)','ka'), 
+('ger(B)|deu(T)','de'), 
+('gre(B)|ell(T)','el'), 
+('grn','gn'), 
+('guj','gu'), 
+('hat','ht'), 
+('hau','ha'), 
+('heb','he'), 
+('her','hz'), 
+('hin','hi'), 
+('hmo','ho'), 
+('hun','hu'), 
+('ice(B)|isl(T)','is'), 
+('ido','io'), 
+('ibo','ig'), 
+('ind','in'), 
+('ina','ia'), 
+('ile','ie'), 
+('iku','iu'), 
+('ipk','ik'), 
+('gle','ga'), 
+('ita','it'), 
+('jpn','ja'), 
+('jav','jv'), 
+('kal','kl'), 
+('kan','kn'), 
+('kau','kr'), 
+('kas','ks'), 
+('kaz','kk'), 
+('kik','ki'), 
+('kin','rw'), 
+('kir','ky'), 
+('kom','kv'), 
+('kon','kg'), 
+('kor','ko'), 
+('kua','kj'), 
+('kur','ku'), 
+('lao','lo'), 
+('lat','la'), 
+('lav','lv'), 
+('lim','li'), 
+('lin','ln'), 
+('lit','lt'), 
+('lub','lu'), 
+('ltz','lb'), 
+('mac(B)|mkd(T)','mk'), 
+('mlg','mg'), 
+('may(B)|msa(T)',''), 
```

Comment:
this is odd that would make it blank

Reply:
looks like should be ms

Reply:
My bad, apparently I missed one... its why I shouldn't merge my own PRs haha.  It should be 'ms'.  I get all the values from here: https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry where I put in the subtag.

Reply:
I'll push another PR to fix it.